src/Directory.Build.props

```diff
@@ -24,7 +24,7 @@
     <!--Framework versions-->
     <UWPTargetPlatformVersion>10.0.14393.0</UWPTargetPlatformVersion>
     <UWPTargetPlatformMinVersion>10.0.14393.0</UWPTargetPlatformMinVersion>
-    <AndroidTargetFramework>monoandroid60</AndroidTargetFramework>
+    <AndroidTargetFramework>monoandroid80</AndroidTargetFramework>
```

Comment:
Is this necessary?

Reply:
Not exacty, but we do need at least `monoandroid70` for `ConstraintLayout`:

![image](https://user-images.githubusercontent.com/3933510/43596894-885e0842-964e-11e8-8770-5a98c4e20395.png)

That said, Microsoft recommends targeting the latest framework for the build (see [here](https://docs.microsoft.com/en-us/xamarin/android/app-fundamentals/android-api-levels?tabs=vswin#target-framework)).  Plus, apps submitted to the Google Play store need to target API 26 (i.e. 8.0) beginning this month (August 2018 - see [here](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html)).